app/src/main/java/com/work/restaurant/view/adapter/BookMarkAdapter.kt

```diff
@@ -0,0 +1,79 @@
+package com.work.restaurant.view.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.work.restaurant.R
+import com.work.restaurant.data.model.FitnessCenterItem
+import com.work.restaurant.view.GlideApp
+import kotlinx.android.synthetic.main.bookmark_item.view.*
+
+class BookMarkAdapter : RecyclerView.Adapter<BookMarkAdapter.ViewHolder>() {
+
+    private val bookmarkArr = ArrayList<FitnessCenterItem>()
```

Comment:
```suggestion
    private val bookmarkArr = ArrayList<FitnessCenterItem>()
```
ArrayList는 Array와 다른 개념이기 때문에
bookmarkArr라는 변수명이 적절하지 않은듯 합니다

Reply:
9b555bf  반영하였습니다